src/qtwebengine/dataprovider.cpp

```diff
@@ -41,8 +97,8 @@ void DataProvider_QWebEngine::requestStarted( QWebEngineUrlRequestJob *request )
     outbuf->close();
 
     // Only delete the buffer when the request is deleted too
-    connect( request, SIGNAL(destroyed()), outbuf, SLOT(deleteLater()) );
+    connect( request, SIGNAL( destroyed() ), outbuf, SLOT( deleteLater() ) );
 
     // We're good to go
-    request->reply( "text/html", outbuf );
+    request->reply( mimetype, outbuf );
```

Comment:
This doesn't work, at least in my tests QtWebEngine simply ignores whatever you pass here.

Reply:
Can I see your code with QtWebEngine?